proto/kvrpcpb.proto

```diff
@@ -1530,3 +1530,37 @@ message TiFlashSystemTableRequest {
 message TiFlashSystemTableResponse {
     bytes data = 1 [(gogoproto.customtype) = "github.com/pingcap/kvproto/pkg/sharedbytes.SharedBytes", (gogoproto.nullable) = false];
 }
+
+// Flush is introduced from the pipelined DML protocol. 
+// A Flush request writes some keys and values to TiKV, storing in LOCK and DEFAULT CF, just like a Prewrite request.
+message FlushRequest {
+    Context context = 1;
+    repeated Mutation mutations = 2;
+    bytes primary_key = 3;
+    uint64 start_ts = 4;
```

Comment:
Shall we name the fields same with the prewrite request?

https://github.com/pingcap/kvproto/blob/932639606bcf3db9676627d55430e1dd10670570/proto/kvrpcpb.proto#L101-L106

Reply:
I'd rather rename the fields in PrewriteRequest. `start_ts` and `primary_key` are more commonly used throughout TiDB and TiKV codebase. What do you think?

Reply:
Yes, because we also have `for_update_ts` and `min_commit_ts`, `start_ts` is a better option.
`primary_lock` only makes sense in prewrite request, `prewrite_key` is easier to understood.

Let's leave the field names in `PrewriteRequest` as a code debt 🙈.